numpy/_core/src/common/npy_pycompat.h

```diff
@@ -34,7 +34,7 @@
         }                                                               \
     }
 #else
-#define NPY_BEGIN_CRITICAL_SECTION_SEQUENCE_FAST(original) {
+#define NPY_BEGIN_CRITICAL_SECTION_SEQUENCE_FAST(original) { do { (void)(original); } while (0)
```

Comment:
needed to suppress unused variable warnings - maybe there's a nicer way to spell this?

Reply:
yeah, I am not sure there is, this seems good.